source/includes/quickstart-settlement.md.erb

```diff
@@ -191,8 +191,57 @@ The investor's investment can be queried by calling:
 
 [GET https://api-sandbox.goji.investments/platformApi/settlement/investors/{clientId}/accounts/{accountType}/investments](/#settlement-debt-get-platformapi-settlement-investors-clientid-accounts-accounttype-investments)
 
+## Repaying the Investment (customers using wallets)
+
+This applies if you are using our flexible payments (wallets) functionality.
+
+> Record Repayment
+
+```json
+{
+  "reference" : "reference",
+  "sourceWalletId": "sourceWalletId",
+  "investorRepayments" : [ {
+    "amount" : {
+      "amount" : 2.68,
+      "currency" : "currency"
+    },
+    "investmentId" : "investmentId",
+    "tax" : {
+      "amount" : 2.68,
+      "currency" : "currency"
+    },
+    "type" : "CAPITAL"
+  }, {
+    "amount" : {
+      "amount" : 2.68,
+      "currency" : "currency"
+    },
+    "investmentId" : "investmentId",
+    "tax" : {
+      "amount" : 2.68,
+      "currency" : "currency"
+    },
+    "type" : "CAPITAL"
+  } ]
+}
+```
+
+[POST https://api-sandbox.goji.investments/platformApi/settlement/investor-repayment](/#settlement-debt-post-platformapi-settlement-investor-repayment)
+
+ - The system will move money from the wallet specified referenced by "sourceWalletId" to all the investors specified in the investorRepayments array.
+ - It is not guaranteed that there will be as many money movements as there are investors in the investorRepayments array so your system should not rely on it.
```

Comment:
Backticks for keywords.

Why is this not guaranteed? Can we be clear on this - just putting my client hat on: surely if they have 10 distinct investors to pay then those distinct investors will get a payment?  Or so we mean if the same investor appears twice they might only get one payment? Or do we mean that there might be fewer movements OUT of the wallet than there are investors (because Receiving)?